src/XRP/model/xrp-payment.ts

```diff
@@ -79,6 +94,7 @@ export default class XRPPayment {
     readonly amount?: XRPCurrencyAmount,
     readonly destination?: string,
     readonly destinationTag?: number,
+    readonly destinationXAddress?: string,
```

Comment:
Can we mark destinationAddress and destinationTag as deprecated?

Reply:
yep, and just to clarify - *don't* mark `account` and/or `sourceTag` deprecated in `XRPTransaction`?  Or should those be marked deprecated too?  I'm trying to understand whether there might be use cases for these objects in which the classic address format might be useful or necessary?

Reply:
Yup I would mark these as deprecated too. Commented as such.